### ISSUE TITLE
Integrate dashboard ideas from Asilomar 2014

### ISSUE THREAD

**Issue by davclark** [BIDS/garmr#2]
from @karthik 
- Where everyone is (Foursquare checkins at select locations and conferences - these could be tagged #msdse to get picked up here, also manual entries). That way it's nice to see who is at what conference. This could be a way to facilitate connections (eg. I see Fernando is at JSM. I know someone else attending who he should speak to)
- Tweets
- Github commits, projects
- New papers (rss)
- Events

We could easily have this together in a day and serve it up from Heroku. Then the page itself could either be chromecast from some local machine onto a spare monitor or to raspberry Pi.

**Comment by kbarbary** [comment]
I have some code to get events from github for a specified set of users (caching results) and parse a subset of them into one-line sentences. E.g.,

![screenshot from 2015-05-15 08 42 30](https://cloud.githubusercontent.com/assets/970135/7656385/fd0e7690-fade-11e4-982e-addcf194a526.png)

Not sure this is what you were thinking, but lets chat.

**Comment by davclark** [comment]
Are you being deliberately provocative by not sharing your code? :P

This is exactly the kind of thing I'd like to see us sharing. My group also wrote code to parse GitHub API data this semester, and so does everyone else.

**Comment by kbarbary** [comment]
Just wrote it yesterday for my own edutainment (like you said, everyone does this...). I wasn't sure where it should go, so my comment was to inquire if it should go in this repo. If you already have something you like for this bit, feel free to ignore.

**Comment by davclark** [comment]
Your output looks awesome. Please stick your code here! I've got some code
that's being re-factored right now in glass-bead-labs/github-bead, but it's
not much to look at now.

On Fri, May 15, 2015 at 9:49 AM, Kyle Barbary notifications@github.com
wrote:

> Just wrote it yesterday for my own edutainment (like you said, everyone
> does this...). I wasn't sure where it should go, so my comment was to
> inquire if it should go in this repo. If you already have something you
> like for this bit, feel free to ignore.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/BIDS/garmr/issues/2#issuecomment-102457122.

## 

Dav Clark
Data Scientist
Berkeley D-Lab + BIDS
bead.glass
917-544-8408

**Comment by karthik** [comment]
I don't understand the main purpose of this repo though.

> A dog from Hel [sic] to guard 190 Doe

Was this repo originally meant to be the ID card scanner?

**Comment by davclark** [comment]
Yeah, maybe we need a new name. I'm pretty open to folks steering the
direction here (modulo bike shedding).

That said, Kelly made a very cute dog image for us.

**Comment by kbarbary** [comment]
Premature logo creation is the root of all evil. ;)

Seriously though... just chatted with @davclark and we decided the dashboard should actually go in a different repo with a different name. This one might still be used for something related to sign-ins.

**Comment by davclark** [comment]
I'm bringing @echiou on-board to work on this stuff. @kbarbary did you ever get around to making the new repo for the dashboard? I want your type-faces.

**Comment by kbarbary** [comment]
It's all at http://github.com/kbarbary/ghdash for the time being. Style is actually just http://bootswatch.com/sandstone/.
